systest/tests/equivocation_test.go

```diff
@@ -31,15 +32,17 @@ func TestEquivocation(t *testing.T) {
 		require.NoError(t, err)
 		keys[i] = priv
 	}
+	// malfeasants := make([]ed25519.PrivateKey, 0, len(keys)-honest)
```

Comment:
Should this be removed?

Reply:
I kinda want to have this kind of assertion - and see that malicious actors are actually marked as such. The problem is that this test is just badly designed for its purpose: two nodes running with the same key act basically as relays for each other.

Because there are so few nodes in this example network and they are well connected they always will have an identical few of the network so they will independently always reach the same results for their ATXs and hare message generation and an equivocation will never happen.

To make this work as intended one of the two nodes running with the same key must either use a different set of PoETs or not receive an ATX by one of its peers (to have a different active set), etc.